groups/prometheus/module-ec2-instances/variables.tf

```diff
@@ -0,0 +1,48 @@
+# Instance
+variable "instance_type" {
+  type = string
+}
+variable "instance_count" {
```

Comment:
Add a single empty line between each variable declaration in this and other variable files to give some clear separation:

```yml
variable "instance_type" {
  ..
}

variable "instance_count" {
  ...
}
```